api/sys_mounts_test.go

```diff
@@ -9,8 +9,34 @@ import (
 	"testing"
 )
 
+func TestGetMount(t *testing.T) {
+	mockVaultServer := httptest.NewServer(http.HandlerFunc(mockVaultGetMountHandler))
```

Comment:
Would it be possible to test getting a real mount from a real / test vault server? I think this would mostly be testing the parsing / decoding logic.

Reply:
Done in https://github.com/hashicorp/vault/pull/25499/commits/63344e0b1b3d46383ea29014c3a98cc7c7538268